src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
-- include: install_latest.yml
+- include_tasks: install_latest.yml
   when: rabbitmq_version == '3.10.13-1'
 
-- include: install_old.yml
+- include_tasks: install_old.yml
   when: rabbitmq_version == '3.10.7-1'
 
```

Comment:
the tasks are dynamic because their inclusion depends on the value of rabbitmq_version.